templates/inventario/proveedorCrear.html

```diff
@@ -0,0 +1,53 @@
+{% extends 'base.html' %}
+
+
+
+{% block content %}
+
+<div class="container" style="background-color:#FEF0F0;">
+    <div class="row">
+        <div class="col-md-6 col-md-offset-3">
+            <div class="panel-default panel">
+                <div class="panel-heading" style="background-color:#FEF0F0;">
+                    <div class="row mb_20">
+                        <div class="col-xs-6">
+                            <a class="active">Añadir Proveedor</a>
+                        </div>
+                    </div>
+                </div>
+                <div class="panel-body" style="background-color:#FEF0F0;">
+                    <div class="row">
+                        <div class="col-lg-12">
+                            <form id="formulario-proveedor" action="#" method="post">
+                                {% csrf_token %}
+
+                                <div class="form-group">
+                                    <label for="inputNitProveedor">Nit del proveedor</label>
+                                    <input id="inputNitProveedor" name="nitProveedor" type="text" class="form-control">
+                                </div>
+                                <div class="form-group">
+                                    <label for="inputDireccionProveedor">Dirección del proveedor</label>
+                                    <input id="inputDireccionProveedor" name="direccionProveedor" type="text" class="form-control">        
+                                </div>
+                                <div class="form-group">
+                                    <label for="inputTelefonoProveedor">Teléfono del proveedor</label>
+                                    <input id="inputTelefonoProveedor" name="telefonoProveedor" type="text" class="form-control">        
```

Comment:
no se verifica que el telefono sea un numero valido, se puede hacer agregando esto al modelo. :D
telefono_regex = RegexValidator(regex=r'^\+?1?\d{7,10}$', message="El telefono debe tener formato: '+7777777'. Up to 10 digits allowed.")
    telefono = models.CharField(validators=[telefono_regex], max_length=12, blank=True) # validators should be a list